### DISCUSSION TITLE
Site limit control not working or lagging behind

### DISCUSSION THREAD

**Discussion by Macgregorsound** [WillCodeForCats/solaredge-modbus-multi#645] [General]
### Describe the bug

When changing the site limit of the setup, the active power limit goes to 90% and the system does not listen to the site limit value. Sometimes after 2 hours it will respond and start working as expected.

### Expected behavior

Site limit is set from 55000W to 200W max export, the system should then start to run in zero-export mode (solar keeping up with internal consumption)

### Diagnostic File

[config_entry-solaredge_modbus_multi-683527698ba1cdcabcb907ad3651c797.json](https://github.com/user-attachments/files/16479373/config_entry-solaredge_modbus_multi-683527698ba1cdcabcb907ad3651c797.json)

### Home Assistant Version

2024.7.4

### solaredge-modbus-multi Version

2.4.18

### Installation Type

HAOS

### Read the Instructions

- [X] My configuration follows https://github.com/WillCodeForCats/solaredge-modbus-multi/wiki/Configuration
- [X] I have read the Known Issues https://github.com/WillCodeForCats/solaredge-modbus-multi/wiki/Known-Issues
- [X] This issue is not about a template error (do not report template errors, read https://github.com/WillCodeForCats/solaredge-modbus-multi/wiki/Template-Design-Notes)

**Comment by cyberkryten** [comment]
I think to change the Site Limit (solaredgemodbus_i1_site_limit) you need to send a "Commit" and that effectively resets the inverter.  It isn't a viable thing to change regularly because of this as it takes 1-2min to come back online (on my system at least).

If you want to stop the system generating/exporting energy, then the way to do that is by using solaredgemodbus_i1_storage_discharge_limit as that doesn't require a restart of the inverter and takes effect quickly.

**Reply by cralwalker** [reply]
my HA entity 'number.solaredge_i1_storage_discharge_limit' is not available. Has it changed identity?

**Reply by cralwalker** [reply]
My retailer remotely curtail export at negative FiT, by the look of it by dropping Site Limit to 0. If each change incur a reset, @cyberkryten, I will reach out to the tech team at the retailer to change their approach, if I can make it work myself first.


<img width="986" alt="image" src="https://github.com/user-attachments/assets/fcffd746-5f76-46ee-901b-47ca8695f7a6">
